spec/lib/zxcvbn_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+require "#{Rails.root}/lib/zxcvbn_tester"
```

Comment:
what if we moved this file to `spec/support/` so it gets auto-required? (see spec/rails_helper.rb:20)